Library/Homebrew/cli/parser.rb

```diff
@@ -54,7 +54,7 @@ def switch(*names, description: nil, env: nil, required_for: nil, depends_on: ni
         end
         process_option(*names, description)
         @parser.on(*names, *wrap_option_desc(description)) do
-          enable_switch(*names, from: :args)
+          enable_switch(*names.last, from: :args)
```

Comment:
Is this needed as well as the change below?

Reply:
turns out it is not necessary. removed.